PeapodSwipe/AuthViewController.swift

```diff
@@ -55,152 +53,89 @@ class AuthViewController: UIViewController {
+                            })
+                    case 201:
+                        if let bearerToken = response.response?.allHeaderFields["Authorization"] as? String {
+                            //print(bearerToken)
+                            let setting = UserInfo(email: email, token: bearerToken, inviteCode: inviteCode, isLoggedIn: true, skipIntro: false)
+                            let key: String = String(describing: UserInfo.self)
+                            UserDefaults.standard.df.store(setting, forKey: key)
+                            Auth.auth().createUser(withEmail: email, password: "ppod9ppod9", completion: { (_, error) in
+                                //print("Sign In error: \(String(describing: error))")
+                                if error == nil {
+                                    Analytics.logEvent("sign_up", parameters: [ "email": email, "invite_code": inviteCode ])
+                                    Instabug.identifyUser(withEmail: email, name: email)
+                                    Instabug.setUserAttribute(inviteCode, withKey: "Invite Code")
+                                    let cardViewController = CardViewController()
+                                    let controller = UINavigationController(rootViewController: cardViewController)
+                                    return self.present(controller, animated: true, completion: nil)
+                                }
+                            })
+                        }
+                    default:
+                        self.showOKAlert(title: "Something Went Wrong",
+                            message: "Please re-enter your email and invite code.")
+                    }
+                }
+        } //end Alamofire.request
     }
 
-    func showRetryMessage() {
-        let alert = UIAlertController(
-            title: "Something Went Wrong",
-            message: "Please re-enter your email and invite code.",
-            preferredStyle: .alert
-        )
-        alert.addAction(
-            UIAlertAction(
-                title: NSLocalizedString("OK", comment: "Ok"),
-                style: .cancel,
-                handler: nil
-            )
-        )
-        self.present(alert, animated: true)
-    }
+    // Stack Overflow: https://stackoverflow.com/questions/25471114/how-to-validate-an-e-mail-address-in-swift#25471164
+    private func isValidEmail(emalAddress: String) -> Bool {
+        let emailRegEx = "[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,64}"
 
-    @objc func SELSignInWithEmail() {
+        let emailTest = NSPredicate(format: "SELF MATCHES %@", emailRegEx)
+        return emailTest.evaluate(with: emalAddress)
+    }
+    private func isDebugMode() -> Bool {
+        let args = ProcessInfo.processInfo.arguments
+        return (UserDefaults.standard.bool(forKey: "FASTLANE_SNAPSHOT") || args.contains("UI_TEST_MODE"))
```

Comment:
The purpose for those flags are not for debugging, they are for running UI Tests, and generating Marketing Screenshots

Reply:
Updated to ```private func isTestMode() -> Bool { ... }```